packages/core/svelte/src/workSpace.ts

```diff
@@ -0,0 +1,14 @@
+import { writable } from 'svelte/store';
+import type { Program, Provider, web3 } from '@project-serum/anchor';
+import type { Connection, Keypair } from '@solana/web3.js';
+import type { Idl } from '@project-serum/anchor';
+
+type WorkSpace = {
+	baseAccount?: Keypair;
+	connection: Connection;
+	provider?: Provider;
+	program?: Program<Idl>;
+	systemProgram?: typeof web3.SystemProgram;
+};
+
+export const workSpace = writable<WorkSpace>(undefined);
```

Comment:
Not sure what this is, but it doesn't belong in the core package, not even as a devDependency. This is for wallets only, not Solana programs or Anchor.

Reply:
This is now moved to the `svelte-ui` package

Reply:
I've removed this file and moved to #162 There will be necessary not only for Anchor, but even with it to share the connection in the whole application.

Reply:
Sorry, no Anchor dependency, including in the UI package. I'm a big fan of Anchor, but it's not a dependency in this project, the goal of which is wallet support. Feel free to publish a library of helper utilities if you like!

Reply:
Put another way, if I'm building a dapp on Solana with Svelte and I'm not using Anchor, I should be able to use all the packages I need without downloading, importing, or expecting Anchor as a dependency. Utilities that support Anchor without adding a dependency (e.g. the `useAnchorWallet` hook) are okay though.

Reply:
ah ok, that makes sense! This was our take on the `useAnchorWallet`. So since we are just using it as a type. Would you be open to having wider typing? That's on the verge of `any` or `unknown`

Reply:
I'm not sure I follow. How is defining a workspace with an IDL similar to `useAnchorWallet`?

Reply:
Sorry, there's some missing context. There's an `AnchorConnectionProvider.svelte` that goes with this in the `svelte-ui` package. Fully agree with your earlier point on the anchor dep. So we'll move that to a separate package all together